wdl/pipelines/TechAgnostic/Utility/CleanupIntermediate.wdl

```diff
@@ -27,18 +28,34 @@ task CleanupAFolder {
     input {
         String bucket_name
         String submission_id
+        Boolean keep_logs = false
     }
 
+    Boolean fail = submission_id == ""
     command <<<
-        timeout 23h gsutil -q rm -rf gs://~{bucket_name}/submissions/~{submission_id} || echo "Timed out. Please try again."
+        if ~{fail}; then exit 1; fi
+        if ~{keep_logs}; then
+            # keep some (presumably) lightweight cromwell log files
+            timeout 23h \
+            gcloud -q storage ls --recursive \
+                gs://~{bucket_name}/submissions/~{submission_id} \
+                | grep -v "*log$" | grep -vF '/stderr' | grep -vF '/stdout' | grep -vF '/script' \
+                | gcloud -q storage rm -rf -I \
+            || echo "Timed out. Please try again."
+        else
+            timeout 23h \
+            gcloud -q storage rm \
```

Comment:
Just wondering, the workflow says cleanupIntermediate files which makes me think it will delete everything except the workflow outputs. But this command would delete the submission folder which would include the output. Does this mean its not advised to run this task unless the outputs of the submission are saved elsewhere?